docs/concepts/extensions.rst

```diff
@@ -0,0 +1,256 @@
+.. _extensions:
+
+Aspects Extensions
+#####################
+
+Aspects is a project that comes with a set of extensions that can be used to
+extend the different parts of the project.
+
+Superset Assets
+================
+
+Aspects allows to programatically define custom superset assets via a tutor patch.
+Developers can use an tutor inline plugin with the patch `superset-extra-assets`
+and those will be imported at initialization.
+
+An example of such a plugin is the following:
+
+.. code-block:: yaml
+
+    name: custom-inline-plugin
+    version: 0.1.0
+    patches:
+    superset-extra-assets: |
+        - _file_name: my-dashboard.yaml
+        dashboard_title: "..."
+        ...
+        - _file_name: my-chart.yaml
+        slice_name: "..."
+        ...
+        - _file_name: my-database.yaml
+        database_name: "..."
+        ...
+        - _file_name: my-dataset.yaml
+        table_name: "..."
+        ...
+
+The patch is expected to be a list of assets with an extra attribute called **_file_name**,
+which uniquely identifies the asset entry. This file does not need to exist anywhere;
+it will be created with the rest of the yaml in that stanza as part of the init process.
```

Comment:
```suggestion
it will be created with the rest of the yaml in that stanza as part of the Tutor / Aspects initialization process.
```

Reply:
There's a lot of good content in here, but I think that we should probably make this page a list of the extension points with a short summary of why someone would want to use them, then move each of these sections to a how-to doc that can be linked to from here. These seem more technical than a high level concept doc.